src/monattack.cpp

```diff
@@ -1194,8 +1194,9 @@ bool mattack::science( monster *const z ) // I said SCIENCE again!
 
             // if the player can see it
             if( g->u.sees( *z ) ) {
-                add_msg( m_warning, _( "The %s shudders, and some sort of caustic fluid leaks from a its damaged shell!" ),
-                z->name().c_str() );
+                add_msg( m_warning,
```

Comment:
This looks unrelated and is a change made in #28569. Is this intentional?

Reply:
Astyle fail in Gorgon forced me to astyle this bit.